spikeinterface/sorters/runsorter.py

```diff
@@ -164,116 +156,112 @@ def run_sorter_docker(sorter_name, recording, docker_image, output_folder=None,
     (parent_folder / 'in_docker_params.json').write_text(
         json.dumps(check_json(sorter_params), indent=4), encoding='utf8')
     # the py script
+
     py_script = f"""
 import json
 from spikeinterface import load_extractor
 from spikeinterface.sorters import run_sorter_local
 
 # load recorsding in docker
-recording = load_extractor('/sorting_output_folder/in_docker_recording.json')
+recording = load_extractor('{parent_folder}/in_docker_recording.json')
```

Comment:
@samuelgarcia why do we use `parent_folder` instead of `output_folder` here?

Reply:
because the output_folder do no exists yet it will created inside the container. Here we are before.